backend/service/topology/topology.go

```diff
@@ -66,6 +66,11 @@ func New(cfg *any.Any, logger *zap.Logger, scope tally.Scope) (service.Service,
 		scope:  scope,
 	}, nil
```

Comment:
nit: can you clean this up actually, should just be c := &client{...} then return c, nil

weird to allocate your own nil err

Reply:
Idk what i was thinking here, i think previously this was just a return statement